apps/openmw/mwgui/tradeitemmodel.cpp

```diff
@@ -24,6 +24,18 @@ namespace MWGui
         return mItems[index];
     }
 
+    ItemModel::ModelIndex TradeItemModel::getIndex(ItemStack item)
+    {
+        size_t i = 0;
+        for (std::vector<ItemStack>::iterator it = mItems.begin(); it != mItems.end(); ++it)
+        {
+            if (*it == item)
+                return i;
+            ++i;
+        }
+        return -1;
```

Comment:
Why not just return std::find() - std::begin(), or -1 if not found? IMO this would make it more obvious what is happening here.